src/main.cpp

```diff
@@ -153,7 +154,20 @@ static std::optional<std::string> choose_socket(wl_display *display)
 
 static wf::config_backend_t *load_backend(const std::string& backend)
 {
-    auto [_, init_ptr] = wf::get_new_instance_handle(backend);
+    std::string config_plugin(backend);
+
+    if (backend.size())
+    {
+        /** Get the full path if the given name is not a full path */
+        if (backend.compare(0, 1, "/") != 0)
```

Comment:
This is also already done by the plugin-loader, I'd say we should move this in the get_plugin_path_for_name() function. Maybe also mention in the docstring that it accepts full paths.

Reply:
Done